SCRWebService/endpoints/gp_summary_upload.py

```diff
@@ -0,0 +1,39 @@
+import os
+import json
+import tornado.web
+import tornado.ioloop
+import scr.gp_summary_update as scr_update
+import utilities.integration_adaptors_logger as log
+
+logger = log.IntegrationAdaptorsLogger('SCR_WEB')
```

Comment:
Here you've got `SCR_WEB`, in `main.py` you've got `SCR-WEB`. Perhaps just make this one more specific.